shared-libs/contacts/src/places.js

```diff
@@ -75,6 +75,12 @@ const validatePlace = place => {
     if (!_.isString(place.contact) && !_.isObject(place.contact)) {
       return err(`Property "contact" on place ${placeId} must be an object or string.`);
     }
+    if (_.isObject(place.contact)) {
+      const err = people._validatePerson(place.contact);
+      if (err) {
+        return err(err);
```

Comment:
When using a contact format:
```
 "contact": {
    "name": "Paula",
    "phone": "+254883720652"
  }
```
At this point we get a `Server error: TypeError: err is not a function`
Could you please improve the way the error is being handled?

Reply:
oh i see the issue, let me fix that